README.md

```diff
@@ -1,5 +1,13 @@
 # Shippo API Python wrapper
 
+---
+
+:warning: **We do not actively maintain these libraries** <br>
+However, you can use them as a starting point and update them for your needs and future changes to our APIs.<br>

```

Comment:
However, you can use it as a starting point and update it for your needs and future changes to our APIs.